README.md

```diff
@@ -1754,6 +1755,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Tung Pham](https://github.com/TungPhamQ)
   - [Tushar Pathak](https://github.com/tushar54594)
   - [Twinkle](https://github.com/twinkletrivaan)
+ 
```

Comment:
Please remove this extra line.
```suggestion
```

Reply:
The extra line remove. Can you review it again, please?